### ISSUE TITLE
Missing libssl-dev dependency

### ISSUE THREAD

**Issue by GustavoLR548** [drwhut/tabletop_club_lobby_server#1]
Hi, just letting you know that I have been trying to install this signalling server recently, and I faced the following issue when running ```cargo build --release´``

```sh
error: failed to run custom build command for `openssl-sys v0.9.103`
```

I later found out that's because I needed to download the ```libssl-dev``` package to my ubuntu. I don't know if this is the standard for other distributions, but I thought it would be interesting to mention in the readme since it is required to actually build the server.

**Comment by drwhut** [comment]
Yeah, this is normal for using the OpenSSL library in Rust. From what I recall, the error message actually says to install your distribution's ssl-dev library, but I will consider adding a note in the README about this as well.
